package/CHANGELOG

```diff
@@ -13,7 +13,7 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/?? IAlibay, ianmkenney, PicoCentauri
+??/??/?? IAlibay, ianmkenney, PicoCentauri, richardjgowers
```

Comment:
Odd, I thought you had conttributed in 2.6 for the GSD stuff, maybe that was 2.5... time flies by.